hypervisor/arch/x86/boot/cpu_primary.S

```diff
@@ -216,7 +216,15 @@ jmpbuf_32:
     /* 0x0008 = HOST_GDT_RING0_CODE_SEL */
     .word 0x0008
 
-.code64
+    .org 0x200
+    .code64
+    .global cpu_primary_start_64
+
+cpu_primary_start_64:
+    lea boot_regs(%rip), %rax
+    movl %edi, (%rax)
+    movl %esi, 4(%rax)
+
```

Comment:
RFC: At the end I would like to squash all commits into single commit. But still is it better to separate a commit for HV from the one for efi-stub? I mean, we could have two commits at the end.

Reply:
I'll let the maintainers chime in but in my opinion, it's preferable to have small commits that are organized somewhat logically (by components, etc) as opposed to one big commit that gets squashed.